ci-operator/step-registry/origin/e2e-gcp-sdn/origin-e2e-gcp-sdn-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: origin-e2e-gcp-sdn
+  steps:
+    pre:
+    - chain: ipi-install
```

Comment:
@pecameron this won't enable any of the SDN options unless you add a `- ref: sdn-conf` here

Reply:
fixed